src/parsers/urdf.hpp

```diff
@@ -58,21 +58,6 @@ namespace se3
                        Model & model, 
                        const bool verbose = false) throw (std::invalid_argument);
 
-    ///
-    /// \brief Build the model from a URDF file with a particular joint as root of the model tree.
-    ///
-    /// \param[in] filemane The URDF complete file path.
-    /// \param[in] rootJoint The joint at the root of the model tree.
-    /// \param[in] verbose Print parsing info.
-    ///
-    /// \return The se3::Model of the URDF file.
-    ///
-    PINOCCHIO_DEPRECATED
```

Comment:
Maybe here we should keep the deprecated method until v1.3. I am not sure. We are not very rigorous with that. I certainly will not complain.